### ISSUE TITLE
Epic: V4.x Tools to support JS EMS engine and update dependencies

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#2101]
The Javascript engine has been moving from the traditional CommonJS engine to the EMS engine for many new features. Several of our engine-dependent libraries (such as Puppeteer) support only EMS in their new versions. Therefore, our tools have to move to the EMS.

### Hill 
Users will be able (1) to use new libraries that require the EMS; (2) to use our improved accessibility tools due to the use of the latest dependent libraries and a better EMS engine.  

### Playback Demonstrations
- Each sprint replay will use [rules showcase template](https://ibm.box.com/s/idg9kyep9gw3gshecifx92smefum3fjh) to explain the issue, including user impact, supporting standard(s), what's broken, fix, help, mapping, and test case(s) to show the issue and the fix.    

### Metric
- Number of rules and help files that are fixed and merged. The [Engine rules change summary2024 ](https://ibm.ent.box.com/notes/1116814404939) is updated after each new or fixed rule, help, or Engine change. 
 - New Checker versions deployed to the stores also includes the [Release Notes](https://github.com/IBMa/equal-access/releases) that includes the list of fixes and features that users read.

   
### Comms
- The team's overall release notes posted to Slack, Office Hours, other developer meetings, etc. will contain the engine changes, especially the new [Checker Rule Set deployment dates](https://www.ibm.com/able/requirements/checker-rule-sets) that will be an integral part of the communications, such as the Rule Set Options in the Checker Settings. 

### Issues assigned to this Epic:
1. Merge the master to the V4.x branch: #2111 
2. Update the NodeJs and other dependencies to the latest
3.  Fix issues related to EMS 
4.  Unit test

## What project/section is the Epic for?

- [Checker](https://github.ibm.com/IBMa/E2E/issues#workspaces/end-to-end-overview-5c90ee20e000f050ae43c45e/board?labels=checker%20-%20internal%20only)

## Definition of Done

- [x] ~~Results are added to the Release Mural with a description of what was accomplished, KPIs, etc.~~
- [x] All content has been reviewed for grammar, spelling, style and technical accuracy
- [x ] All code committed to main branch
- [x] All unit tests pass (including accessibility)
- [x] Demo works in staging environment
- [ ] At least one performance metric / monitor constructed

### Lessons learned and observations
v3 and v4 parallel deployment for customer's try-on period.
